docs/registry/advanced/create-subtype.md

```diff
@@ -18,7 +18,7 @@ You can define a new {{< glossary_tooltip term_id="resource" text="resource" >}}
 
 {{% alert title="Tip" color="tip" %}}
 
-If you want to use most of an existing API but need just a few other functions, try using the [`DoCommand`](/appendix/apis/#docommand) endpoint and [extra parameters](/sdks/use-extra-params/) to add custom functionality to an existing subtype.
+If you want to use most of an existing API but need just a few other functions, try using the `DoCommand` endpoint and [extra parameters](/sdks/use-extra-params/) to add custom functionality to an existing subtype.
```

Comment:
no link?

Reply:
Do you have a suggestion of where do link? There is no general #docommand anywhere. It's folded into each subtype's docs. So I think this will have to do

Reply:
good w that